myadmin-service-api/src/main/java/za/ac/unisa/myadmin/generic/dto/GenericCodeInfo.java

```diff
@@ -0,0 +1,69 @@
+package za.ac.unisa.myadmin.generic.dto;
+
+import java.io.Serializable;
+
+/**
+ * Created by Adrian on 2018-07-27.
+ */
+public class GenericCodeInfo implements Serializable {
```

Comment:
Yeah agree I also disliked naming it like that nothing came to mind at the time.
So coupled tightly to the table name hence Entity and dto naming.

@trojanc suggested dropping Generic think I'll go that route. 
Would you rename the entity class as well?

Reply:
Good question. I know one can name the entity anything and then slab on @Table reference to the DB table, plus I lean toward naming the entities as close as possible to the real table name. Again with those stupid table names it can be difficult to judge what to call the entity. On top of that what if there is a Code table also?
I vote for leaving the entity name as is and perhaps make the info dto object name more descriptive. Or add some Java doc class description to the code.

Reply:
I agree, the entity name luckily only lives far behind the services...if it has a less attractive name there, atleast it only exist until it gets converted to an API InfoObject with a proper name

Reply:
I agree as well. So dto's get renamed.

Reply:
> Again with those stupid table names it can be difficult to judge what to call the entity
Recently noticed they give more description in comments of some tables.
e.g
select * from all_tab_comments where TABLE_NAME = 'GENCOD';
"OWNER",	"TABLE_NAME","TABLE_TYPE","COMMENTS"
"VOSDATAD","GENCOD","TABLE","Generic Code"